anvio/__init__.py

```diff
@@ -28,6 +28,25 @@
 AS_MARKDOWN = '--as-markdown' in sys.argv
 FIX_SAD_TABLES = '--fix-sad-tables' in sys.argv
 DOCS_PATH = os.path.join(os.path.dirname(__file__), 'docs')
+TMP_DIR = None
+
+# if the user wants to use a non-default tmp directory, we set it here
+if '--tmp-dir' in sys.argv:
+    try:
+        idx = sys.argv.index('--tmp-dir')
+        TMP_DIR = os.path.abspath(sys.argv[idx+1])
+
+        if not os.path.exists(TMP_DIR):
+            os.makedirs(TMP_DIR)
```

Comment:
if there is no directory AND not permission to create one, this statement will result an error.

Reply:
I added [this](https://github.com/merenlab/anvio/pull/1691/commits/648d28c46f814ec3aa5b4cd11e6275a15c6907ea).

Is it better? :)

Reply:
```
[ ] NOT NAILED / [X] NAILED 
```